getting-started.md

```diff
@@ -0,0 +1,142 @@
+## How to install and run Vega Wallet
+These instructions are written to be used in command line. Below, in the snippets, you'll see commands in blue text. Copy those instructions and paste them into your command line interface.
```

Comment:
I think have the filename in uppercase, such as `GETTING_STARTED.md`, is better to be consistent with the README.m and to make it a bit more visible to the users coming to the repo. What do you think ?

Reply:
Sure, happy to make that change.